src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -82,7 +82,7 @@ internal sealed class Options : TransformInputBase
         /// <summary>
         /// The columns passed to this <see cref="ITransformer"/>.
         /// </summary>
-        public IReadOnlyCollection<(string outputColumnName, string inputColumnName)> Columns => ColumnPairs.AsReadOnly();
+        internal IReadOnlyCollection<(string outputColumnName, string inputColumnName)> Columns => ColumnPairs.AsReadOnly();
```

Comment:
>internal [](start = 8, length = 8)

why make them internal? the user might want to inspect them?

Reply:
This one is specifically called out in the issue #2827 (item 5 in the list). @Ivanidzo4ka can you elaborate?